ocs_ci/utility/kms.py

```diff
@@ -888,7 +888,7 @@ def validate_external_vault(self):
             raise NotFoundError("KMS flag not found")
 
     def create_vault_csi_kms_token(
-        self, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+        self, namespace=config.ENV_DATA["cluster_namespace"]
```

Comment:
Same as above, change the default value to `None` and set it to the default from config in the code also here please.